.github/workflows/pull_request.yaml

```diff
@@ -47,7 +47,7 @@ jobs:
   tests:
     name: Run Tests
     runs-on: ubuntu-latest
-    container: node:16
+    container: node:16-bullseye
```

Comment:
Node 16 reached its End of Life. I think we should check if it compiles OK and passes our tests on node 18/20/22 and upgrade the flows and Dockerfile.